docs/vtex-io/Reference/vtex-io-documentation-vtex-io-cli-installation-and-command-reference/vtex-io-documentation-vtex-io-cli-command-reference.md

```diff
@@ -2,65 +2,66 @@
+| [`publish`](#publish)                                 | Publishes the app in the current directory as a release candidate version.                                                      |
+| [`release`](#release)                                 | (For git users only.) Bumps the app version, commits, and pushes to the remote repository for the app in the current directory. |
+| [`setup`](#setup)                                     | Sets up typing and tools for the current development environment.                                                               |
+| [`switch`](#switch)                                   | Switches to another VTEX account.                                                                                               |
+| [`undeprecate`](#undeprecate)                         | Reverts a deprecated version of an app to a stable version.                                                                     |
+| [`uninstall`](#uninstall)                             | Uninstalls an app from the current account and workspace.                                                                       |
+| [`unlink`](#unlink)                                   | Unlinks an app from the current workspace.                                                                                      |
+| [`update`](#update)                                   | Updates all installed apps to the latest minor or patch version. Does not upgrade to another major version.                     |
+| [`whoami`](#whoami)                                   | Prints details about the current account, workspace, environment, and login information.                                        |
+| [`workspace abtest finish`](#workspace-abtest-finish) | Stops all A/B tests on the current account.                                                                                     |
+| [`workspace abtest start`](#workspace-abtest-start)   | Starts a new A/B test on the current workspace.                                                                                 |
+| [`workspace abtest status`](#workspace-abtest-status) | Displays the results of the active A/B tests.                                                                                   |
+| [`workspace delete`](#workspace-delete)               | Deletes one or many workspaces from the current account.                                                                        |
+| [`workspace list`](#workspace-list)                   | Lists all workspaces of the current account.                                                                                    |
+| [`workspace promote`](#workspace-promote)             | Promotes the current workspace to the master branch. Only applicable for production workspaces.                                 |
```

Comment:
```suggestion
| [`workspace promote`](#workspace-promote)             | Promotes the current workspace to master. Only applicable for production workspaces.                                 |
```

Reply:
The `master `in here is for the master workspace concept in VTEX IO: https://developers.vtex.com/docs/guides/vtex-io-documentation-workspace